GOVERNANCE.md

```diff
@@ -57,39 +60,60 @@ decision on a matter.
 
 While this approach has served us well, as the Project grows and faces more
 legal and financial decisions and interacts with other institutions, we see a
-need for a more formal governance model. Moving forward The Project leadership
-will consist of a BDFL and Steering Council. We view this governance model as
-the formalization of what we are already doing, rather than a change in
-direction.
+need for a more formal governance and organization model.
+We view this governance model as the formalization of what we are already doing,
+rather than a change in direction.
+
+## Community and Team Architecture
+The PyMC community is organized in an onion-like fashion.
+The tiers relevant to the project governance are listed below sorted by
+increasing responsibility. Due to the onion-like structure, members of a group are
+also members of all the groups listed above:
+
+* Contributors
+* Recurring Contributors
+* Core Contributors
+* Steering Council
```

Comment:
Does the steering council have a min/fixed/max size?

Reply:
I'd set some constraints, but I wasn't really sure which ones. Sorry I'm a bit all over the place, this is from the PR description:

> council membership constraints
>
> I think we should have a minimum number of members, say 5, with at least one member per team and at most 2 institutional contributors of the same company. Or some other numbers but some kind of constraints on those ends. I don't care much about the numbers really, but I think some constraints would be good to have

Reply:
Trying to be more specific, one proposal could be:

* between 4-7 members
* at least 1 member per team
* at most 2 institutional members per company